shared/constants/tokens.js

```diff
@@ -15,6 +15,7 @@ export const LISTED_CONTRACT_ADDRESSES = Object.keys(contractMap).map(
  * @property {string} address - The address of the selected 'TOKEN' or
  *  'NFT' contract.
  * @property {string} [symbol] - The symbol of the token.
+ * @property {string|null} [name] - The name of the token.
```

Comment:
What is the reason for updating this JSDoc?

Reply:
Whoa, TIL, I didn't know files can import JSDoc.

Reply:
I did not know that either ^^; in the sendFlow the name of the NFT was not being displayed 
<img width="613" alt="Screenshot 2025-03-20 at 11 16 44" src="https://github.com/user-attachments/assets/44532980-ff80-47c7-8a8c-f2c896e82271" />

In the UI we try to display `asset.symbol`, so i'm adding here a fallback to `asset.name` if symbol does not exist. 

The incoming asset if of type `Asset` from `'../../../ducks/send'` which was missing the optional `name` in `TokenDetails`. Without updating the jsDoc, lint would complain when i try to do
`symbol: (asset.details.symbol || asset.details.name)`